coralproject/talk
# Load settings in front end

## What does this PR do?

Loads settings to front end and reflects pre and post mod behavior.

## How do I test this PR?

1. Enable premoderation.
1. Load embed stream.
2. Redux store should have a config object which includes a premod setting.
3. Post a comment. No comment should appear, and you should receive a notice saying that the comment is being reviewed.
4. Approve the coment in the admin screen and reload the page. The comment should appear.
5. Disable permoderation.
6. Reload the page and post a comment. The comment should appear immediately.

@coralproject/tech

… behavior.
